priv/script.sh

```diff
@@ -2,441 +2,490 @@
 
 set -eu
 
+elixir_version=1.17.3-otp-27
+erlang_version=27.1.2
+phoenix_version=1.7.14
+postgres_version=15.1
+
+OS="$(uname -s)"
```

Comment:
This has been *partially* moved upwards to improve `$HOME` resolution. See below.